distrho/src/DistrhoPluginJack.cpp

```diff
@@ -512,6 +523,13 @@ class PluginJack
     }
 #endif
 
+#if DISTRHO_PLUGIN_WANT_MIDI_INPUT
```

Comment:
sendNoteCallback is only defined if using a plugin with midi input, which means the constructor will fail on a plugin with different configuration :P

Reply:
Oops. That wasn't very good :p